src/components/Capture/capture.js

```diff
@@ -85,29 +85,40 @@ class Capture extends Component {
     this.validateCapture(message.id, valid)
   }
 
-  handleBase64 = (lossyBase64, base64) => {
-    const blob = base64toBlob(base64)
-    this.handleImage(lossyBase64, blob, base64)
-  }
-
-  handleImage = (base64ImageLossy, image, base64) => {
-    if (!image) {
+  handleImage = (lossyBase64, blob) => {
```

Comment:
Ignore, I think `handleImage` is OK. I think `handleImageInternal` can be improved - it does not need to be a function if it's just the implementation of `handleImage`